.github/workflows/ci-generated.yml

```diff
@@ -0,0 +1,37 @@
+name: Generate Files
+
+on:
+  push:
+      branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    # Check out master, so that we can count.
+    - uses: actions/checkout@v2
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: "3.7"
+    - name: Generate Files
+      # Record the count in a tmp file so it survives changing branches
+      run: |
+        make init-ci && make all
+        sed -i.bak '/*_generated.rs/d' .gitignore
+    - name: Commit files
+      run: |
+        git add .
+        git config --local user.email "action@github.com"
+        git config --local user.name "GitHub Action"
+        git commit -m "Update Generated Files" -a
```

Comment:
`"Add Generated Files"` maybe?
given there was no generated files in the forcibly pushed history.